File: BeerTier/client/src/components/Beer.js
import { Card, CardContent, Typography } from "@mui/material";

export default function Beer({ beer }) {
  return (
    <Card className="beer__card">
      <CardContent>
        <Typography variant="h4">{beer.name}</Typography>
        {beer.styles.map((style) => {
          return <div>#{style.name}</div>;
        })}
        <Typography variant="h5">
          {beer.brewery.name} ({beer.category.name})
        </Typography>
        <p>{beer.content}</p>
        <Typography variant="h6">
          posted by {beer.userProfile.displayName} on {beer.createDateTime}
        </Typography>
      </CardContent>
    </Card>
  );
}

File: BeerTier/Models/Beer.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace BeerTier.Models
{
    public class Beer
    {
        public int Id { get; set; }

        [Required]
        [MaxLength(55)]
        public string Name { get; set; }
        public string Content { get; set; }

        [DataType(DataType.Url)]
        [MaxLength(255)]
        public string ImageLocation { get; set; }

        [Required]
        public int BreweryId { get; set; }

        [Required]
        public DateTime CreateDateTime { get; set; }

        [Required]
        public int UserProfileId { get; set; }

        public Brewery Brewery { get; set; }
        public List<Style> Styles { get; set; }
        public UserProfile UserProfile { get; set; }
        public List<Comment> Comments { get; set; }
    }
}
